nf_core/list.py

```diff
@@ -374,13 +374,13 @@ def get_local_nf_workflow_details(self):
                         self.active_tag = str(tag)
 
             # I'm not sure that we need this any more, it predated the self.branch catch above for detacted HEAD
-            except TypeError as e:
+            except (TypeError, git.InvalidGitRepositoryError) as e:
                 log.error(
-                    "Could not fetch status of local Nextflow copy of {}:".format(self.full_name)
-                    + "\n   {}".format(str(e))
-                    + "\n\nIt's probably a good idea to delete this local copy and pull again:".format(self.local_path)
-                    + "\n   rm -rf {}".format(self.local_path)
-                    + "\n   nextflow pull {}".format(self.full_name)
+                    f"Could not fetch status of local Nextflow copy of '{self.full_name}':"
+                    f"\n   [red]{type(e).__name__}:[/] {str(e)}"
+                    "\n\nIt's probably a good idea to delete this local copy and pull again:"
```

Comment:
I think it would be helpful to explain why we recommend deleting and repulling. Something like the following:
```suggestion
                    "\n\nThis git repository looks broken. It's probably a good idea to delete this local copy and pull again:"
```